Civi/API/Kernel.php

```diff
@@ -145,7 +146,7 @@ public function runRequest($apiRequest) {
       }
       catch (\Civi\API\Exception\UnauthorizedException $e) {
         // We catch and re-throw to log for visibility
-        \CRM_Core_Error::backtrace('API Request Authorization failed', TRUE);
+        \Civi::log()->error('API Request Authorization failed: ' . $apiRequest->getEntityName() . '::' . $apiRequest->getActionName(), ['apiRequest[where]' => $apiRequest->getWhere()]);
```

Comment:
@mattwire the getWhere here is an issue it seems presumably because API3 calls will come through here as well as v4?

```
api_v4_OAuthSysTokenTest::testLimitedApiAccess
CRM_Core_Exception: Unknown api parameter: getWhere

/home/homer/buildkit/build/build-3/web/sites/all/modules/civicrm/Civi/Api4/Generic/AbstractAction.php:235
````